Assets/MRTK/Providers/WindowsMixedReality/Shared/WindowsMixedRealityHandMeshProvider.cs

```diff
@@ -3,24 +3,28 @@
 
 using Microsoft.MixedReality.Toolkit.Input;
 using Microsoft.MixedReality.Toolkit.Utilities;
+
+#if WINDOWS_UWP
 using System;
 using System.Threading.Tasks;
 using Unity.Profiling;
 using UnityEngine;
-
-#if WINDOWS_UWP
 using Windows.Perception.People;
 using Windows.UI.Input.Spatial;
 #endif // WINDOWS_UWP
 
 namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality
 {
     /// <summary>
-    /// Defines the additional data, like hand mesh, that an articulated hand on HoloLens 2 can provide.
+    /// Queries the hand mesh data that an articulated hand on HoloLens 2 can provide.
     /// </summary>
-    public class WindowsMixedRealityArticulatedHandDefinition : ArticulatedHandDefinition
+    public class WindowsMixedRealityHandMeshProvider
```

Comment:
Is this considered a breaking change and if it is should we add it to the release notes?

Reply:
Ah, yeah the class no longer existing would be a breaking change, though it shouldn't really have any use outside of these classes. I'll re-add and obsolete.

Reply:
Re-added!